sous-chef/sous_chef/menu/create_menu.py

```diff
@@ -78,18 +81,18 @@ def get_menu_for_grocery_list(
     ) -> (list[MenuIngredient], list[MenuRecipe]):
         self.load_local_menu()
 
-        manual_ingredient_list = (
-            self.dataframe[self.dataframe["type"] == "ingredient"]
-            .apply(self._retrieve_manual_menu_ingredient, axis=1)
-            .tolist()
-        )
+        entry_funcs = {
+            "ingredient": self._retrieve_manual_menu_ingredient,
```

Comment:
It could stay as it is, but what about moving this to a function. It could be that one exists that could be adapted to handle this.

Reply:
Yes, thought about this too, I think it would be better, I'll take a look.